server/plugin/api.go

```diff
@@ -56,6 +56,18 @@ type PRDetails struct {
 	Reviews            []*github.PullRequestReview `json:"reviews"`
 }
 
+type FilteredNotification struct {
+	github.Notification
+	HTMLUrl string `json:"html_url"`
```

Comment:
```suggestion
	HTMLURL string `json:"html_url"`
```
You might have done this to differentiate between the two words and increase readability but I think Go suggests to always use caps for abbreviation
see what suits best here as per Go